package.json

```diff
@@ -65,11 +65,11 @@
     "nyc": "^15.1.0",
     "p-queue": "^6.6.2",
     "prom-client": "^14.2.0",
+    "quickjs-emscripten": "^0.23.0",
```

Comment:
This works, but I am wondering if we should continue our Rust porting work and instead use something like https://crates.io/crates/boa_engine

Reply:
I've started an effort in hs/boajs.